csharp/extractor/Semmle.Extraction.CSharp/Kinds/TypeAnnotation.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+namespace Semmle.Extraction.Kinds
+{
+    [Flags]
```

Comment:
Where is it used like a flag set in the code?

Reply:
It isn't, currently, however it is "morally" a set and could be used in that way.